### ISSUE TITLE
UselessParentheses false positive with arrow functions (PHP 7.4)

### ISSUE THREAD

**Issue by xthiago** [slevomat/coding-standard#726]
Starting with PHP 7.4 we'll have the ability to create [arrow functions](https://wiki.php.net/rfc/arrow_functions_v2). The stable version will be available at the end of the year, but right now I'm testing the alpha2 release.

An arrow function with only one argument triggers a unwanted error `Useless parentheses (SlevomatCodingStandard.PHP.UselessParentheses.UselessParentheses)`:

```php
$firstName = 'John';
$name  = fn($firstName) => 'FirstName: ' . $firstName;
```

Zero or two or more arguments does not trigger an error - as expected. 

```php
// arrow function without arguments
$fullName = fn() => 'John Doe';

// arrow function with two or more arguments
$firstName = 'John';
$lastName  = 'Doe';
$fullName  = fn($firstName, $lastName) => $firstName . $lastName;
```

Environment:

- PHP: `PHP 7.4.0alpha2 (cli) (built: Jul 12 2019 01:56:19)`
- phpcs: `PHP_CodeSniffer version 3.4.2 (stable)`
- slevomat/coding-standard: `5.0.4`

**Comment by kukulich** [comment]
Not possible without https://github.com/squizlabs/PHP_CodeSniffer/issues/2523

**Comment by adaamz** [comment]
looks that the phpcs is closed now

**Comment by kukulich** [comment]
Fixed in master.
